### ISSUE TITLE
All Pages - High priority A - Skip link

### ISSUE THREAD

**Issue by gingerbenw** [datasciencecampus/sdg-indicators#1057]
![screen shot 2017-11-02 at 10 20 12](https://user-images.githubusercontent.com/24570483/32320945-6ddd3bdc-bfb7-11e7-9a08-614906f1710f.png)

The skip link does not work in Internet Explorer, as the keyboard user continues to tab after selecting, focus returns to the top of the page.  This is an issue in Internet Explorer.  

**Code Ref(s):** `<div id="main-content" class="container goal-tiles" role="main">`

**Comment by gingerbenw** [comment]
**Solution:** Ensure that a tabindex=”-1” is used on the destination element so that focus remains and does not return the user to the top of the page.   Suggested code: <div id="main-content" class="container goal-tiles" role="main" tabindex=”-1”>
